release/release_tests.yaml

```diff
@@ -344,3 +344,471 @@
     file_manager: job
 
   alert: xgboost_tests
+
+#######################
+# Tune cloud  tests
+#######################
+- name: tune_cloud_aws_no_sync_down
```

Comment:
Btw, can you document the API somewhere later? It seems like we have many new ones.

Reply:
The config API should be documented in the top of the file in an extensive comment